client/keystore/src/local.rs

```diff
@@ -278,24 +278,29 @@ impl SyncCryptoStore for LocalKeystore {
 	}
 
 	fn has_keys(&self, public_keys: &[(Vec<u8>, KeyTypeId)]) -> bool {
-		public_keys.iter().all(|(p, t)| self.0.read().key_phrase_by_type(&p, *t).is_ok())
+		public_keys.iter()
+			.all(|(p, t)| self.0.read().key_phrase_by_type(&p, *t).ok().flatten().is_some())
 	}
 
 	fn sr25519_vrf_sign(
 		&self,
 		key_type: KeyTypeId,
 		public: &Sr25519Public,
 		transcript_data: VRFTranscriptData,
-	) -> std::result::Result<VRFSignature, TraitError> {
+	) -> std::result::Result<Option<VRFSignature>, TraitError> {
 		let transcript = make_transcript(transcript_data);
 		let pair = self.0.read().key_pair_by_type::<Sr25519Pair>(public, key_type)
 			.map_err(|e| TraitError::PairNotFound(e.to_string()))?;
```

Comment:
IMO this error no longer makes sense, if the pair wasn't there we'd be returning `None` from `key_pair_by_type` right?